hypershift-operator/controllers/scheduler/aws/dedicated_request_serving_nodes.go

```diff
@@ -262,6 +263,9 @@ func (r *DedicatedServingComponentScheduler) Reconcile(ctx context.Context, req
 			// If subnets are separated by periods, replace them with commas
 			lbSubnets = strings.ReplaceAll(lbSubnets, ".", ",")
 		}
+		if node.Labels[OSDFleetManagerPairedNodesLabel] != "" && pairLabel == "" {
+			pairLabel = node.Labels[OSDFleetManagerPairedNodesLabel]
```

Comment:
How could be different from `pairLabel == ""` if you set it up  as empty un 254?
Is this being reset in every reconciliation cycle?

Reply:
I used the same pattern as for the variables above. And i think it's a small optimization to set it only once, even if there are multiple nodes (because we iterate over nodesToUse.